aws/database/rds/vpc.tf

```diff
@@ -0,0 +1,47 @@
+locals {
+  vpc_cidr_block = var.vpc_cidr_block
```

Comment:
I think we can remove this local variable and use `var.vpc_cidr_block` directly (instead of `local.vpc_cidr_block`)